src/sentry/notifications/notifications/rules.py

```diff
@@ -96,20 +97,31 @@ def get_subject(self, context: Mapping[str, Any] | None = None) -> str:
     def reference(self) -> Model | None:
         return self.group
 
+    def get_unsubscribe_key(self) -> tuple[str, int, str | None] | None:
+        return "issue", self.group.id, None
+
     def get_recipient_context(
         self, recipient: RpcActor, extra_context: Mapping[str, Any]
     ) -> MutableMapping[str, Any]:
         timezone = pytz.timezone("UTC")
 
+        unsubscribe_key = self.get_unsubscribe_key()
+        unsubscribe_link = None
+
         if recipient.actor_type == ActorType.USER:
             user_tz = UserOption.objects.get_value(user=recipient, key="timezone", default="UTC")
             try:
                 timezone = pytz.timezone(user_tz)
             except pytz.UnknownTimeZoneError:
                 pass
+            if unsubscribe_key:
+                key, resource_id, referrer = unsubscribe_key
+                unsubscribe_link = get_unsubscribe_link(recipient.id, resource_id, key, referrer)
+
```

Comment:
i kinda thought get_unsubscribe_key would make it added to the context

Reply:
or at least it seems like it
https://github.com/getsentry/sentry/blob/master/src/sentry/mail/notifications.py#L92-L98

Reply:
`get_context` is redefined in `AlertRuleNotification` though :/ so that one never gets called